api/v1alpha1/kinds.go

```diff
@@ -0,0 +1,38 @@
+package v1alpha1
+
+import "k8s.io/apimachinery/pkg/runtime"
+
+const (
+	// KindPodChaos is the kind for pod chaos
+	KindPodChaos = "PodChaos"
+	// KindNetworkChaos is the kind for network chaos
+	KindNetworkChaos = "NetworkChaos"
+	// KindIOChaos is the kind for io chaos
+	KindIOChaos = "IoChaos"
+	// KindKernelChaos is the kind for kernel chaos
+	KindKernelChaos = "KernelChaos"
+	// KindStressChaos is the kind for stress chaos
+	KindStressChaos = "StressChaos"
+	// KindTimeChaos is the kind for time chaos
+	KindTimeChaos = "TimeChaos"
+)
+
+var (
+	// Kinds is a collection to keep chaos kind name and its type
+	Kinds = map[string]*ChaosKind{
```

Comment:
I don't if this is okay. But it is very dangerous since var is changeable during runtime.

Reply:
If we use this way, why not register these values in different files.